test/25_service-installs.ts

```diff
@@ -212,18 +229,10 @@ export default () => {
 							ctx.app2Service1.id,
 						);
 					});
-					await expectNewTasks(
+					await expectNewSettledTasks(
 						'create_service_installs',
 						isServiceInstallEnabled
 							? [
-									// the first one is from unpinning
```

Comment:
Rather than commenting why we expect 2 tasks here, I moved each one right after the action that creates them.